src/to_curl.rs

```diff
@@ -170,6 +170,31 @@ pub fn translate(args: Cli) -> Result<Command> {
         cmd.push("--key");
         cmd.push(keyfile.to_string_lossy());
     }
+    if let Some(Some(tls_version)) = args.ssl {
+        match tls_version {
+            tls::Version::TLS_1_0 => {
+                cmd.push("--tlsv1");
```

Comment:
nit: can we use `--tlsv1.0` here instead of `--tlsv1`?

Reply:
I think I did that because `--tlsv1` has existed for much longer. But `--tls-max` postdates both and `--tlsv1` on its own is wrong. So `--tlsv1.0` is better.